lib/plugin.js

```diff
@@ -37,7 +37,11 @@ class LazyCompilePlugin {
     this.options = Object.assign(
       {
         refreshAfterCompile: false,
-        ignores: [/html-webpack-plugin/],
+        ignores: [
```

Comment:
These rules should not be included in the plugin. We should pass it in options:

```js
new LazyCompileWebpackPlugin({
	ignores: [
      /\bhtml-webpack-plugin\b/,
      /\b(html|raw|to-string)-loader\b/,
      /\bexports\.toString\b/
    ]
})
```

Reply:
@liximomo Just thought about that either, is `/\bhtml-webpack-plugin\b/` should be included by default always?

Reply:
And `raw-loader` should be included by default too in my opinion which emits plain output like `module.exports = ...`

Reply:
We need to ignore a loader only if it is used with an entry file. (always true for `html-webpack-plugin`).

Reply:
@liximomo Please review again to check whether the description for `ignore` option is correct and suitable.